autoload/go/list.vim

```diff
@@ -151,6 +151,7 @@ let s:default_list_type_commands = {
       \ "GoTest":               "quickfix",
       \ "GoVet":                "quickfix",
       \ "GoReferrers":          "quickfix",
+      \ "GoImplements":         "quickfix",
```

Comment:
The default for `GoImplements` should be `locationlist`, because the information that it the list will contain is relative to a single buffer.